docs/Plugins.rst

```diff
@@ -765,29 +765,32 @@ and displayed with::
 
 The available settings are:
 
-+----------------+---------+---------------------------------------+
-| Setting        | Default | Description                           |
-+================+=========+=======================================+
-| blocks         | true    | Allow blocks, boulders, logs, or bars |
-+----------------+---------+ to be matched for generic "building   |
-| boulders       | true    | material" items                       |
-+----------------+---------+                                       |
-| logs           | true    |                                       |
-+----------------+---------+                                       |
-| bars           | false   |                                       |
-+----------------+---------+---------------------------------------+
-| quickfort_mode | false   | Enable compatibility mode for the     |
-|                |         | legacy Python Quickfort (not required |
-|                |         | for DFHack quickfort)                 |
-+----------------+---------+---------------------------------------+
++----------------+---------+-----------+---------------------------------------+
+| Setting        | Default | Persisted | Description                           |
++================+=========+===========+=======================================+
+| all_enabled    | false   | no        | Enable planning mode for all building |
+|                |         |           | types.                                |
++----------------+---------+-----------+---------------------------------------+
+| blocks         | true    | yes       | Allow blocks, boulders, logs, or bars |
++----------------+---------+           | to be matched for generic "building   |
+| boulders       | true    |           | material" items                       |
++----------------+---------+           |                                       |
+| logs           | true    |           |                                       |
++----------------+---------+           |                                       |
+| bars           | false   |           |                                       |
++----------------+---------+-----------+---------------------------------------+
```

Comment:
I didn't know this worked - cool!
![image](https://user-images.githubusercontent.com/3719547/106237453-df268b80-61cc-11eb-8a41-7da0d987b724.png)